apps/patient/src/views/Pharmacy.tsx

```diff
@@ -757,45 +756,31 @@ export const Pharmacy = () => {
               <HStack>
                 <Text whiteSpace="nowrap">Sort by</Text>
                 <HStack w="full">
-                  <Button
-                    w="50%"
-                    size="lg"
-                    isActive={sortBy === 'price'}
-                    _active={{
-                      backgroundColor: 'brand.500',
-                      color: 'white',
-                      borderColor: 'brand.500'
-                    }}
-                    border="2px"
-                    borderColor="gray.100"
-                    backgroundColor="white"
-                    onClick={() => setSortBy('price')}
-                    borderRadius="xl"
-                  >
-                    Cash Price
-                  </Button>
-                  <Button
-                    w="50%"
-                    size="lg"
-                    isActive={sortBy === 'distance'}
-                    _active={{
-                      backgroundColor: 'brand.500',
-                      color: 'white',
-                      borderColor: 'brand.500'
-                    }}
-                    border="2px"
-                    borderColor="gray.100"
-                    backgroundColor="white"
-                    onClick={() => setSortBy('distance')}
-                    borderRadius="xl"
-                  >
-                    Distance
-                  </Button>
+                  {(['distance', 'price'] as SortBy[]).map((sort) => (
```

Comment:
note: you actually want `as const` rather than `as SortBy[]`. 

If you add a 3rd `SortBy` or remove one of them, this will still typecheck. But if you use `as const` and `distance` or `price` is no longer in `SortBy` then it'll typefail